unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx

```diff
@@ -158,19 +163,29 @@ export const WithdrawFundModal = ({
               <div className="flex flex-col gap-2 leading-tight text-md text-brand-dark">
                 <Detail label="Network:">{networkName}</Detail>
                 <Detail label="Amount to transfer:">{`${amountToTransfer} ${symbol}`}</Detail>
-                <Detail label="Beneficiary:">{beneficiary}</Detail>
-                {isContract && (
-                  <p className="text-red-500">
-                    This is a contract address, please make sure this contract
-                    can handle the funds, or they will be lost.
-                  </p>
-                )}
-                {!isContract && noBalance && (
-                  <p className="text-red-500">
-                    This address does not seem to have been used on{' '}
-                    {`${networkName}`}
-                    before, please ensure it is correct or funds will be lost.
-                  </p>
+                <Detail label="Beneficiary">{beneficiary}</Detail>
+                {isLoading ? (
+                  <>
```

Comment:
I would wrap it in Placeholder.Root component. This way, we can control spacing in future.